backend/settings.js

```diff
@@ -18,6 +18,7 @@ const DEFAULT_SETTINGS = {
     PLOTLY_API_DOMAIN: 'api.plot.ly',
 
     // TODO - Should this be configurable or should it just be a constant?
+    CONNECTOR_HOST_INFO: {},
```

Comment:
what's the shape of this? could we expand it out into separate variables so that it's easier to understand?